lib/plugins/inspec-plugin-manager-cli/lib/inspec-plugin-manager-cli/cli_command.rb

```diff
@@ -102,16 +103,17 @@ def search(search_term) # rubocop: disable Metrics/AbcSize
       #==================================================================#
       #                       inspec plugin install
       #==================================================================#
-      desc "install [-v VERSION] PLUGIN", "Installs a plugin from rubygems.org, a gemfile, or a path to local source."
+      desc "install [options] PLUGIN", "Installs a plugin from rubygems.org, a gemfile, or a path to local source."
       long_desc <<~EOLD
-        PLUGIN may be the name of a gem on rubygems.org that begins with inspec- or train-.
+        PLUGIN may be the name of a gem on rubygems.org or an alternate source that begins with inspec- or train-.
```

Comment:
It's possible that this could be confused to mean that the alternate source must be begin with `inspec-`

I suggest 

```
PLUGIN may be the name of a gem on rubygems.org (or an alternate source) that begins with `inspec-` or `train-`.
```

or simply remove mention of the source here and have

```
PLUGIN is the name of a gem that begins with `inspec-` or `train-`.
```

Reply:
I picked the first one, as both the `desc` and the following line in the `long_desc` create a tension betweet mentioning only rubygems.org and mentioning local installs. I think, leaving out the reference to external sources would confuse readers as well.